### PR TITLE
scylla_setup: enable node_exporter for offline installation

### PR DESCRIPTION
**Pull request by amoskong**
node_exporter had been added to scylla-server package by commit
95197a09c91eb719c252abed847b8c035f5f9600.
    
So we can enable it by default for offline installation.

/CC @tzach @slivne @eyalgutkind @syuu1228

### CODE REVIEW

**Comment by amoskong** [comment]
This PR had been tested by: 

- https://jenkins.scylladb.com/view/master/job/scylla-master/job/reproducers/job/byo-artifact-centos7-test/3/console
  - test package: http://scratch.scylladb.com/amos/scylla-unified-package-4.4.dev.v2.20201224.72c2482f7.tar.gz

```
[scylla-test@artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 system]$ sudo systemctl status scylla-node-exporter
● scylla-node-exporter.service - Prometheus exporter for machine metrics
   Loaded: loaded (/usr/lib/systemd/system/scylla-node-exporter.service; enabled; vendor preset: disabled)
  Drop-In: /etc/systemd/system/scylla-node-exporter.service.d
           └─dependencies.conf
   Active: active (running) since Fri 2020-12-25 03:43:09 UTC; 4min 2s ago
 Main PID: 4793 (node_exporter)
   CGroup: /scylla.slice/scylla-helper.slice/scylla-node-exporter.service
           └─4793 /opt/scylladb/node_exporter/node_exporter --collector.interrupts

Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=thermal_zone
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=time
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=timex
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=udp_queues
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=uname
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=vmstat
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=xfs
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.043Z caller=node_exporter.go:112 collector=zfs
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.045Z caller=node_exporter.go:191 msg="Listenin...=:9100
Dec 25 03:43:09 artifacts-centos7-jenkins-db-node-aeeb8d12-0-1 node_exporter[4793]: level=info ts=2020-12-25T03:43:09.045Z caller=tls_config.go:170 msg="TLS is disa...=false
Hint: Some lines were ellipsized, use -l to show in full.
```

File: dist/common/scripts/scylla_setup [hunk 1]
```diff
@@ -193,30 +193,30 @@ if __name__ == '__main__':
...
-                default_no_coredump_setup = True
-                warn_offline_missing_pkg('coredump setup', 'systemd-coredump')
-
-            if is_debian_variant() and not shutil.which('cpufreq-set'):
-                default_no_cpuscaling_setup = True
-                warn_offline_missing_pkg('cpuscaling setup', 'cpufrequtils')
-            elif not shutil.which('cpupower'):
-                default_no_cpuscaling_setup = True
-                warn_offline_missing_pkg('cpuscaling setup', 'cpupower')
+    if not is_nonroot():
```
Comment (syuu1228) [inline dist/common/scripts/scylla_setup:196]: 1. There is no description why we need to remove `if is_offline():`
2. If we remove `if is_offline():`, normal package users (who installed scylla via apt / yum) will see these offline mode messages, it must be wrong.

**Review by syuu1228 — CHANGES_REQUESTED** [review]

**Comment by syuu1228** [comment]
"scylla_setup: enable node_exporter for offline installation" part looks OK.

File: dist/common/scripts/scylla_setup (hunk 1 above)
Comment (amoskong) [reply dist/common/scripts/scylla_setup]: Fixed.

**Comment by amoskong** [comment]
> "scylla_setup: enable node_exporter for offline installation" part looks OK.

I had updated the second patch to add is_offline() back, please help to review, Thx.

**Review by syuu1228 — APPROVED** [review]
LGTM

**Pull request merged**